content/docs/intro/cloud-providers/digitalocean/setup.md

```diff
@@ -0,0 +1,31 @@
+---
+title: Setup
```

Comment:
Same as previous comment regarding SEO-friendly page title- e.g. "DigitalOcean Pulumi Provider Setup and Configuration".